index.html

```diff
@@ -4,29 +4,28 @@
   <head>
     <meta charset="utf-8">
     <title>140 Or Less</title>
-      <link rel= "stylesheet" href="styles.css">
+      <link rel="stylesheet" href="style.css">
+      <link href="https://fonts.googleapis.com/css?family=Reem+Kufi:300,400,700" rel="stylesheet">
+      <link rel="stylesheet" href="path/to/font-awesome/css/font-awesome.min.css">
```

Comment:
This `href` for Font Awesome is incorrect. `path/to/font-awesome/css/font-awesome.min.css` is just an example; you need to use a real URL. You can get one from a CDN.